torch/_inductor/codegen/cuda/gemm_template.py

```diff
@@ -478,35 +588,67 @@ def render(  # type: ignore[override]
         self,
         kernel: CUDATemplateKernel,
         op: "cutlass_gemm_op.GemmOperation",  # type: ignore[name-defined]
-        output_node: Optional[Buffer] = None,
+        template_node: Optional[Buffer] = None,
+        epilogue_nodes: Optional[List[IRNode]] = None,
     ) -> str:
+        if epilogue_nodes is not None and len(epilogue_nodes) > 0:
```

Comment:
Why is there a difference between `len(epilogue_nodes) > 0` and `len(epilogue_nodes) > 1`?

Reply:
I assume you mean a difference between len(epilogue_nodes)==0 and len(epilogue_nodes) > 0 ? In that case we need to prepare a bit of information required by cutlass_epilogue_gen to generate the EVT code.

Reply:
I see, good catch, that's a typo, should be > 0